vaadin-date-picker-overlay.html

```diff
@@ -18,17 +19,32 @@
       }
 
       #toolbar {
-        padding: 1em;
+        padding: 0.3em;
         background: #eee;
         @apply(--shadow-elevation-2dp);
+        @apply(--layout-horizontal);
       }
 
       #scroller {
         height: 100%;
       }
+
+      #yearLabl {
```

Comment:
"yearLabl" seems like a typo. Make it either "yearLabel" or "yearLbl". I prefer the first option. I would also make "todayBtn" into "todayButton".